eq-publisher/src/eq_schema/builders/routing2/basicQuestionnaireJSON.js

```diff
@@ -0,0 +1,141 @@
+const questionnaireJson = {
+  id: "1",
+  title: "Basic Questionnaire JSON",
+  description: "",
+  theme: "default",
+  legalBasis: "StatisticsOfTradeAct",
+  navigation: false,
+  surveyId: "",
+  summary: false,
+  metadata: [],
+  sections: [
+    {
+      id: "1",
+      title: "<p>Section 1</p>",
+      introductionTitle: null,
+      introductionContent: null,
+      introductionEnabled: false,
+      pages: [
+        {
+          id: "1",
+          title: "<p>Page 1</p>",
+          description: "",
+          guidance: null,
+          pageType: "QuestionPage",
+          routingRuleSet: null,
+          confirmation: null,
+          answers: [
+            {
+              id: "1",
+              type: "Currency",
+              label: "Answer 1",
+              description: "",
+              guidance: "",
+              properties: { decimals: 0, required: false },
+              qCode: "",
+              validation: {
```

Comment:
Can we remove these validations as they aren't required for any of the routing and makes this questionnaire more complex.